server/komodo-rest/src/main/java/org/komodo/rest/KomodoRestV1Application.java

```diff
@@ -177,11 +197,6 @@
         String DATA_SERVICE_SEGMENT = "dataservice"; //$NON-NLS-1$
 
         /**
-         * The name of the URI path segment for DataService clone in the Komodo workspace.
-         */
-        String CLONE_DATA_SERVICE_SEGMENT = "clone"; //$NON-NLS-1$
-
-        /**
```

Comment:
Not sure you mean by that.
The reason this is being removed is to add the CLONE_SEGMENT constant at L177. That constant is now not clear which service it relates, unlike the other SEGMENT constants.